rest-api/src/test/java/com/cloudbees/workflow/rest/endpoints/JobAndRunAPITest.java

```diff
@@ -62,8 +62,8 @@ public void test() throws Exception {
                 "   stage ('Test'); " +
                 "   echo ('Testing'); " +
                 "   stage ('Deploy'); " +
-                ( hudson.Functions.isWindows() ? "   bat ('rmdir /s/q targs || echo no such dir\\n mkdir targs && echo hello> targs\\\\hello.txt'); "
-                                               : "   sh ('rm -rf targs && mkdir targs && echo hello > targs/hello.txt'); "
+                (hudson.Functions.isWindows() ? "   bat ('rmdir /s/q targs || echo no such dir\\n mkdir targs && echo hello> targs\\\\hello.txt'); "
+                        : "   sh ('rm -rf targs && mkdir targs && echo hello > targs/hello.txt'); "
```

Comment:
BTW you can use steps like `deleteDir` & `writeFile` to avoid a platform dependency.